cmake/Corrosion.cmake

```diff
@@ -548,6 +558,26 @@ function(corrosion_set_env_vars target_name env_var)
     )
 endfunction()
 
+function(corrosion_set_cargo_flags target_name)
+    # corrosion_set_cargo_flags(<target_name> [FLAGS <flag1> ... ])
+    set(options)
+    set(one_value_args)
+    set(multi_value_args FLAGS)
+    cmake_parse_arguments(
+            SET
+            "${options}"
+            "${one_value_args}"
+            "${multi_value_args}"
+            "${ARGN}"
+    )
```

Comment:
Just remove the whole part with `cmake_parse_arguments`.
The signature can just be `corrosion_set_cargo_flags(<target_name> <flag1> [.... <flagN>])`
Below in the set_property call you can just use `${ARGN}` instead of `${SET_FLAGS}`